### ISSUE TITLE
Update Bellota & Bellota Text

### ISSUE THREAD

**Issue by kemie** [google/fonts#8239]
**Font Project Git Repo URL:**

https://github.com/kemie/Bellota-Font

**Current version:**

4.1

**New available version:**

5

**Super short description of the upgrade:**

Expanded the weight range to go from thin to black. Variable versions available. Lots of improvements to letterforms & spacing. 

**Bugs reported in this repo which this upgrade would fix:**

**Comment by ravid-dubsteper** [comment]
Љ and ђ
![image_2024-10-02_17-14-35](https://github.com/user-attachments/assets/b9e0adfd-948f-4b09-83ed-9a88bcc4cb44)

**Comment by vv-monsalve** [comment]
HI @kemie, thanks for the upgrade request. I'm adding it now for the projects to be assigned.

**Comment by ravid-dubsteper** [comment]
> Hi @ravid-dubsteper, could you please clarify what you are pointing at with the comment?

Hello, I just showed the bugs in the font
The middle vertical stroke of the Љ is thicc and the descender of the rest of the ђ aren't combined
Also ң has the same problem
![image](https://github.com/user-attachments/assets/df5508de-eeaa-4668-a47c-08a7b8b4905e)

**Comment by vv-monsalve** [comment]
Before onboarding the font, the person in charge will always work with the designers on the QA revision and all the required adjustments. However, thank you for the early report.

**Comment by kemie** [comment]
@ravid-dubsteper just fixed, thanks!

**Comment by chrissimpkins** [comment]
What is the current blocker on this project?

**Comment by emmamarichal** [comment]
@m4rc1e, I just had a look: these kind of parameters are indeed not well supported by glyphsLib, but I'm pretty sure something can be done with builder2 or some other build script? + Bellota already has been onboarded, I guess we found a way to deal with it before

<img width="1289" height="934" alt="Image" src="https://github.com/user-attachments/assets/174325e8-2c18-4f2a-88f0-d490d579dbd4" />

**Comment by kemie** [comment]
If necessary I could separate into two completely different files, but would make changes more complicated. I seem to remember last time fonts were generated manually.

**Comment by emmamarichal** [comment]
@kemie Yes that would be a solution, but let's try with a custom build script: if we can edit sources in a single place, that's would be great :)

**Comment by kemie** [comment]
Is there anything I could do to help the process along?

**Comment by emmamarichal** [comment]
Hi @kemie! Very sorry for the delay. the project needs some work, I'll have a look next week! :)
